pkg/core/state/notification_event.go

```diff
@@ -20,6 +20,76 @@ type NotificationEvent struct {
 	Item       *stackitem.Array `json:"state"`
 }
 
+type ContractInvocation struct {
+	Hash           util.Uint160     `json:"contract_hash"`
+	Method         string           `json:"method"`
+	Arguments      *stackitem.Array `json:"arguments"`
+	ArgumentsBytes []byte           `json:"arguments_bytes"`
+	ArgumentsCount uint32           `json:"arguments_count"`
+	IsValid        bool             `json:"is_valid"`
```

Comment:
Notice we _never_ use snake_case in JSON structures, this should be something different. Also, from the Go perspective it's very convenient to have the default boolean (`false`) to follow regular use case. Like we use `truncated` for various `find*` results. Maybe `truncated` is applicable here too.